SPECS/auoms/auoms.spec

```diff
@@ -0,0 +1,193 @@
+        echo "Pre: found etc/audit/plugins.d/auoms.conf"
+        if [ -e /etc/audit/plugins.d/auoms.conf.auomssave ]; then
+            rm /etc/audit/plugins.d/auoms.conf.auomssave
+        fi
+        cp -p /etc/audit/plugins.d/auoms.conf /etc/audit/plugins.d/auoms.conf.auomssave
+    fi
+fi
+
+%preun
+#!/bin/sh
+
+if [ $1 -eq 0 ]; then
+    /opt/microsoft/auoms/bin/auomsctl disable
+fi
+
+%post
+#!/bin/sh
+
+SERVICEDIR=/opt/microsoft/auoms
+
+if [ $1 -gt 1 ] ; then
+    if [ -e /etc/audisp/plugins.d/auoms.conf.auomssave ]; then
+        echo "Post: found /etc/audisp/plugins.d/auoms.conf"
+        if [ -e /etc/audisp/plugins.d/auoms.conf ]; then
+            rm /etc/audisp/plugins.d/auoms.conf
+        fi
+        cp -p /etc/audisp/plugins.d/auoms.conf.auomssave /etc/audisp/plugins.d/auoms.conf
+    fi
+    if [ -e /etc/audit/plugins.d/auoms.conf.auomssave ]; then
+        echo "Post: found /etc/audit/plugins.d/auoms.conf"
+        if [ -e /etc/audit/plugins.d/auoms.conf ]; then
+            rm /etc/audit/plugins.d/auoms.conf
+        fi
+        cp -p /etc/audit/plugins.d/auoms.conf.auomssave /etc/audit/plugins.d/auoms.conf
+    fi
+    echo "Post: executing upgrade"
+    /opt/microsoft/auoms/bin/auomsctl upgrade
+fi
+for dir in /usr/lib/systemd/system /lib/systemd/system; do
+    if [ -e $dir ]; then
+        install -m 644 ${SERVICEDIR}/auoms.service $dir
+        systemctl enable auoms.service
+        break
+    fi
+done
+sudo /opt/microsoft/auoms/bin/auomsctl enable
+rm -f /etc/audisp/plugins.d/auoms.conf.*
+rm -f /etc/audit/plugins.d/auoms.conf.*
+
+%postun
+#!/bin/sh
+
+if [ $1 -eq 0 ]; then
+    rm -f /etc/audisp/plugins.d/auoms.conf*
+    rm -f /etc/audit/plugins.d/auoms.conf*
+
+    rm -rf -v /etc/opt/microsoft/auoms
+    rm -rf -v /var/opt/microsoft/auoms
+fi
+for dir in /usr/lib/systemd/system /lib/systemd/system; do
+    if [ -e ${dir}/auoms.service ]; then
+        systemctl disable auoms.service
+        rm -f ${dir}/auoms.service
+        break
+    fi
+done
+
+%files
+%defattr(-,root,root)
+/usr/share/selinux/packages/auoms
+/usr/share/selinux/packages/auoms/*
+%{_sysconfdir}/init.d/auoms
+%{_sysconfdir}/opt/microsoft/auoms
+%{_sysconfdir}/opt/microsoft/auoms/auoms.conf
+%{_sysconfdir}/opt/microsoft/auoms/auomscollect.conf
+%{_sysconfdir}/opt/microsoft/auoms/example_output.conf
+%{_sysconfdir}/opt/microsoft/auoms/outconf.d
+%{_sysconfdir}/opt/microsoft/auoms/rules.d
+/opt/microsoft/auoms
+/opt/microsoft/auoms/LICENSE
+/opt/microsoft/auoms/THIRD_PARTY_IP_NOTICE
+/opt/microsoft/auoms/auoms.service
+/opt/microsoft/auoms/bin
+/opt/microsoft/auoms/bin/auomscollect
+/opt/microsoft/auoms/bin/auoms
+/opt/microsoft/auoms/bin/auomsctl
+/var/opt/microsoft/auoms
+/var/opt/microsoft/auoms/data
+/var/opt/microsoft/auoms/data/outputs
+
+%changelog
+* Thu Oct 22 2020 Andrew Phelps <anphel@microsoft.com> 2.2.5-1
+- Initial CBL-Mariner version.  
+* Tue Sep 15 2020 Henry Li <lihl@microsoft.com> 2.3.0-1
```

Comment:
Can you please change the version number to 2.2.5-1? 2.3.0-1 was taken out from the diagnostics repo and this information was mistakenly not updated.

Reply:
Fixed